amun/base/kustomization.yaml

```
@@ -9,6 +9,8 @@ resources:
   - argo-workflows/inspection-build-with-cpu.yaml
   - argo-workflows/inspection-run.yaml
   - argo-workflows/inspection-run-with-cpu.yaml
+  - argo-workflows/parse-inspection-inputs.yaml
```

```suggestion
  - argo-workflows/create-inspection-complete-message.yaml
```
there is no parse-inspection-inputs.yaml added or available